src/base/index.vue

```diff
@@ -615,8 +752,53 @@ export default {
       });
   },
   methods: {
+    exportSections() {
+      const dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(JSON.stringify(this.allSections));
+      const dlAnchorElem = document.getElementById('downloadAnchorElem');
+      dlAnchorElem.setAttribute("href",     dataStr     );
+      dlAnchorElem.setAttribute("download", `${this.pageName}.json`);
+      dlAnchorElem.click();
+    },
+    initImportSections() {
+      if (Object.keys(this.allSections).length > 0 || this.currentViews.length > 0) {
+        this.showToast(
+            "Warning",
+            "warning",
+            "Import sections only works for empty pages"
+        );
+      } else {
+        this.$refs.jsonFilePick.click();
+      }
+    },
+    importSections(e) {
+      const jsonFile = e.target.files[0];
+      const reader = new FileReader();
+      reader.readAsText(jsonFile, "UTF-8");
+      reader.onload = (evt) => {
+        const jsonFileResult = evt.target.result;
+        const sections = JSON.parse(jsonFileResult);
+        sections.forEach((section) => {
+          if (section.type === "configurable") {
+            this.showToast(
+                "Warning",
+                "warning",
+                "Make sure to activate configurable sections for your project"
```

Comment:
provide a clearer message with the section name to activate.

Also your condition is too light, some configurable section types can be public, in which case the import should pass with no issue. then if the section is a public_scopped or private, you should check if it's been activated already. Because now even if the section is active the warning will show.